### PR TITLE
Feature/sid 168 send token button alway active show error message if needed

### PR DESCRIPTION
**Pull request by sosan-neikbeen**

### CODE REVIEW

File: src/components/partials/profile/ProfileHeader.vue [hunk 1]
```diff
@@ -234,7 +234,13 @@ export default class ProfileHeader extends Vue {
     }
 
     sendToken() {
-        this.sendTokenModalOpen = true;
+        if (this.web3Accounts.length === 0) {
+            this.toast.error(
+                "Add wallet connection with Polkadot.js extension to use this feature"
```
Comment (dobschal) [inline src/components/partials/profile/ProfileHeader.vue:239]: @sosan-neikbeen Please use the following text:
```
To use this feature, please attach a wallet connection with polkadot{.js} extension. Click here for more help.
```

Is it possible to have a link in the toast? 

Would be nice to redirect to: https://parallelfi.gitbook.io/parallel-finance/polkadot-network/parallel-product-guides/a-guide-to-testnet/polkadot-.js-connect-wallet

**Review by dobschal — CHANGES_REQUESTED** [review]
The toast looks nice.

File: src/components/partials/profile/ProfileHeader.vue [hunk 2]
```diff
@@ -206,16 +206,30 @@ export default class ProfileHeader extends Vue {
...
+    toast = useToast();
+
+    toastMessage = h("div", {}, [
+        h(
+            "p",
+            { class: "d-inline" },
+            "To use this feature, please attach a wallet connection with polkadot{.js} extension."
+        ),
+        h("a", {
+            href: "https://parallelfi.gitbook.io/parallel-finance/polkadot-network/parallel-product-guides/a-guide-to-testnet/polkadot-.js-connect-wallet",
```
Comment (dobschal) [inline src/components/partials/profile/ProfileHeader.vue:218]: add ```target="_blank"``` please

**Review by dobschal — CHANGES_REQUESTED** [review]

**Comment by sosan-neikbeen** [comment]
https://user-images.githubusercontent.com/95384931/180190737-f978f495-bb31-4c62-915b-98b29cd58be6.mp4

Toast message example when no web3 account is found

**Review by dobschal — APPROVED** [review]

**Pull request merged**